src/Logary/LogResultModule.fs

```diff
@@ -0,0 +1,38 @@
+namespace Logary
+
+[<CompilationRepresentation(CompilationRepresentationFlags.ModuleSuffix)>]
+module internal LogResult =
+  open Hopac
+
+  let success: LogResult = Alt.always (Result.Ok Promise.unit)
+  let error message : LogResult = Alt.always (Result.Error message)
+
+  let notAcceptByTarget (targetName: string) = error (Message.eventFormat("not accept by {target}", targetName))
+
+module internal LogError =
+  let targetBufferFull (targetName: string): ProcessResult = Result.Error (Message.eventFormat("{target} 's log buffer is full", targetName))
```

Comment:
We don't want to allocate if we're full if we can avoid it. (generally; this is not a comment on having Message values in the Result)